lib/libtrakt/__init__.py

```diff
@@ -1 +0,0 @@
-from trakt import TraktAPI
```

Comment:
Can we rename the library?  libtrakt is obviously a library so the lib prefix is just added noise.